federation/pkg/kubefed/init/init_test.go

```diff
@@ -84,6 +85,7 @@ func TestInitFederation(t *testing.T) {
 			dnsZoneName:        "example.test.",
 			lbIP:               "10.20.30.40",
 			image:              "example.test/foo:bar",
+			etcdPVCapacity:     "10Gi",
```

Comment:
Will be good to have 2 tests: one where you do not specify any value and verify that it picks up 10Gi and one where you specify something other than the default value.